Corgibytes.Freshli.Cli/IoC/FreshliServiceBuilder.cs

```diff
@@ -114,5 +116,9 @@ private void RegisterComputeLibYearCommand()
         Services.AddTransient<IAgentReader, AgentReader>();
     }
 
-    private void RegisterMessageBus() => Services.AddSingleton<MessageBus>();
+    private void RegisterMessageBus()
+    {
+        Services.AddSingleton<MessageBus>();
+        Services.AddHangfire();
```

Comment:
This doesn't work because there's no definition for `AddHangfire`. I've checked out the Hangfire repository, and it's a extension method that should be loaded somehow.